### PR TITLE
statically import methods used from org.junit.Assert

### PR DESCRIPTION
**Pull request by darxriggs**
- that's a common practice so do it consistently in all tests

### CODE REVIEW

File: core/src/test/java/hudson/model/AbstractItemTest.java [hunk 1]
```diff
@@ -3,9 +3,11 @@
  */
 package hudson.model;
 
+import static org.junit.Assert.assertEquals;
+import static org.junit.Assert.assertNull;
+
```
Comment (jglick) [inline core/src/test/java/hudson/model/AbstractItemTest.java]: Or can use a star import. No strong preference.

**Comment by jglick** [comment]
Thanks! BTW there is also an `extends Assert` antipattern, mostly from @kohsuke.

**Comment by tfennelly** [comment]
Fwiw ... I personally dislike static imports of static methods i.e. I personally prefer seeing `Assert.assertEquals` (vs `assertEquals`) in code.

**Comment by jglick** [comment]
I usually agree but make an exception for `Assert`, since otherwise test code becomes unwieldy. But there is no official styleguide for Jenkins.

**Pull request merged**